### PR TITLE
Admin show approve question

### PR DESCRIPTION
**Pull request by phandinhman**

### CODE REVIEW

File: app/models/question.rb [hunk 1]
```diff
@@ -16,6 +16,11 @@ class Question < ActiveRecord::Base
   validates_associated :answers
   validate :validate_answers
 
+  scope :contributed, -> {includes(:user).where(users: {role: 0})}
+  # scope :search, -> q {where "name LIKE ?", "%#{q}%" if q.present?}
```
Comment (hoangmirs) [inline app/models/question.rb]: remove this line and below empty lines

File: app/controllers/admin/questions_controller.rb [hunk 2]
```diff
@@ -28,12 +30,18 @@ def edit
   end
 
   def update
-    if @question.update_attributes question_params
-      flash.now[:success] = t "flash.success.updated_question"
+    if params[:action_type] &&
```
Comment (dieunb) [inline app/controllers/admin/questions_controller.rb]: Dùng strong params đi nào

File: app/controllers/admin/questions_controller.rb [hunk 3]
```diff
@@ -28,12 +30,18 @@ def edit
...
+      @question.update_attributes(status: params[:action_type])
+        flash[:success] = t "flash.success.edit_question",
+          action: t(params[:action_type])
+        redirect_to admin_questions_path
+    elsif @question.update_attributes subject_params
```
Comment (dieunb) [inline app/controllers/admin/questions_controller.rb]: question sao lại dùng `subject_params`?

File: app/views/admin/questions/edit.html.erb [hunk 4]
```diff
@@ -1,3 +1,21 @@
 <% provide :modal_title, t(".edit_question") %>
 <% provide :submit_button, t(".save_button") %>
+<% provide :title, t("title") %>
 <%= render "form" %>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <div class="panel panel-info">
+      <div class="panel-heading">
+      Comfirmation
```
Comment (dieunb) [inline app/views/admin/questions/edit.html.erb]: i18n

**Comment by hoangmirs** [comment]
ok

**Pull request merged**